metron-deployment/development/centos6_docker_build/build_and_run.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+shopt -s nocasematch
+
+function help {
+ echo " "
+ echo "usage: ${0}"
+ echo "    --skip-vagrant-up               skip vagrant up"
+ echo "    --force-docker-build            force build docker machine"
+ echo "    --skip-tags='tag,tag2,tag3'     the ansible skip tags"
+ echo "    -h/--help                       Usage information."
+ echo " "
+ echo "example: to skip vagrant up and force docker build with two tags"
+ echo "   build_and_run.sh -skip-vagrant-up --force-docker-build --skip-tags='solr,sensors'"
+ echo " "
+}
+
+SKIP_VAGRANT_UP=false
+FORCE_DOCKER_BUILD=false
+A_SKIP_TAGS="sensors,solr"
+
+# handle command line options
+for i in "$@"; do
+ case $i in
+ #
+ # SKIP_VAGRANT_UP
+ #
+ #
+  --skip-vagrant-up)
+   SKIP_VAGRANT_UP=true
+   shift # past argument
+  ;;
+
+ #
+ # FORCE_DOCKER_BUILD
+ #
+ #   --force-docker-build
+ #
+   --force-docker-build)
+   FORCE_DOCKER_BUILD=true
+   shift # past argument
+  ;;
+
+ #
+ # SKIP_TAGS
+ #
+ #   --skip-tags='foo,bar'
+ #
+   --skip-tags=*)
+   A_SKIP_TAGS="${i#*=}"
+   shift # past argument=value
+  ;;
+
+ #
+ # -h/--help
+ #
+  -h|--help)
+   help
+   exit 0
+   shift # past argument with no value
+  ;;
+
+ #
+ # Unknown option
+ #
+  *)
+   UNKNOWN_OPTION="${i#*=}"
```

Comment:
Why `#*=`?

Reply:
To make sure we consume the option and value

Reply:
This is kind of boilerplate

Reply:
I would have thought this would only provide back the value, filtering out the option.